pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+addopts = --reuse-db
```

Comment:
This can cause confusion. I think we should remove this.
cc: @annshress

Reply:
Yes, my personal recommendation is commenting this out, and only use this option (when needed) to make tests quicker. 

FYI  https://pytest-django.readthedocs.io/en/latest/database.html#reuse-db-reuse-the-testing-database-between-test-runs

> --reuse-db will not pick up schema changes between test runs.

So comment it out. @k9845